From 70194f53e6c810241ab144d067291bac8ce4ac80 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Tue, 26 Jul 2022 12:14:08 +0300
Subject: [PATCH] [SW-0] gaudi2: sram bist: use modified header name

gaudi2_bist.h header is modified to sram_bist.h
in emb-specs. So changed header name in asic.c file.

Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
Change-Id: Ic6ee8bad7efece54ce1129b5c794006a79539bbf
---

diff --git a/arch/gaudi2/arc/asic.c b/arch/gaudi2/arc/asic.c
index 29f89a1..d2a980f 100644
--- a/arch/gaudi2/arc/asic.c
+++ b/arch/gaudi2/arc/asic.c
@@ -26,7 +26,7 @@
 #include "common.h"
 #include <hl.h>
 #include <errno.h>
-#include <gaudi2_bist.h>
+#include <sram_bist.h>
 #include <memory.h>
 
 #define SDRAM_BASE_FULL 0x1000FFFFFD000000ull
